Can't define struct with a single character identifier.

```
package main
type A struct {
	i i32
}
func main()() {
	var a A
	a.i = 42
	printf("a.i %d\n", a.i)
}
```
output:
```
panic: struct 'A' not found in package 'main'

goroutine 1 [running, locked to thread]:
github.com/skycoin/cx/cxgo/actions.DeclareStruct(0xc420468dc0, 0x1, 0xc420472200, 0x1, 0x1)
	/home/reash/go/src/github.com/skycoin/cx/cxgo/actions/cxgo_compiled.go:211 +0x1b2
github.com/skycoin/cx/cxgo/cxgo0.yyParse(0x952a00, 0xc420488dc0, 0xc420488dc0)
	/home/reash/go/src/github.com/skycoin/cx/cxgo/cxgo0/cxgo0.go:1673 +0xa81
github.com/skycoin/cx/cxgo/cxgo0.Parse(0xc4205b6000, 0x65, 0x64)
	/home/reash/go/src/github.com/skycoin/cx/cxgo/cxgo0/cxgo0.go:74 +0xc8
main.main()
	/home/reash/go/src/github.com/skycoin/cx/cxgo/cxgo.nn.go:8535 +0x3b7f
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -26,6 +26,7 @@
   * #51: Silent name clash between packages
   * #52: Some implicit casts were not being caught at compile time
   * #53: CX was not catching an error involving invalid indirections
+  * #55: Single character declarations are now allowed
   * #131: Problem with struct literals in short variable declarations
 * Documentation
   * CONTRIBUTING.md: Information about how to contribute to CX

--- a/cxgo/cxgo.nex
+++ b/cxgo/cxgo.nex
@@ -785,7 +785,7 @@ func main () {
 					}
 
 					
-					rePkgName := regexp.MustCompile("(^|[\\s])package\\s+([_a-zA-Z][_a-zA-Z0-9]+)")
+					rePkgName := regexp.MustCompile("(^|[\\s])package\\s+([_a-zA-Z][_a-zA-Z0-9]*)")
 
 					if match := rePkgName.FindStringSubmatch(string(line)); match != nil {
 						if pkg, err := cxgo0.PRGRM0.GetPackage(match[len(match) - 1]); err != nil {
@@ -806,7 +806,7 @@ func main () {
 						continue
 					}
 
-					reStrctName := regexp.MustCompile("(^|[\\s])type\\s+([_a-zA-Z][_a-zA-Z0-9]+)?\\s")
+					reStrctName := regexp.MustCompile("(^|[\\s])type\\s+([_a-zA-Z][_a-zA-Z0-9]*)?\\s")
 
 					if match := reStrctName.FindStringSubmatch(string(line)); match != nil {
 						if _, err := cxgo0.PRGRM0.GetStruct(match[len(match) - 1], prePkg.Name); err != nil {
@@ -867,7 +867,7 @@ func main () {
 						continue
 					}
 					
-					rePkgName := regexp.MustCompile("(^|[\\s])package\\s+([_a-zA-Z][_a-zA-Z0-9]+)")
+					rePkgName := regexp.MustCompile("(^|[\\s])package\\s+([_a-zA-Z][_a-zA-Z0-9]*)")
 
 					if match := rePkgName.FindStringSubmatch(string(line)); match != nil {
 						if pkg, err := cxgo0.PRGRM0.GetPackage(match[len(match) - 1]); err != nil {
@@ -920,7 +920,7 @@ func main () {
 						continue
 					}
 					
-					rePkgName := regexp.MustCompile("(^|[\\s])package\\s+([_a-zA-Z][_a-zA-Z0-9]+)")
+					rePkgName := regexp.MustCompile("(^|[\\s])package\\s+([_a-zA-Z][_a-zA-Z0-9]*)")
 
 
 					if match := rePkgName.FindStringSubmatch(string(line)); match != nil {
@@ -943,7 +943,7 @@ func main () {
 						continue
 					}
 
-					reGlblName := regexp.MustCompile("(^|[\\s])var\\s([_a-zA-Z][_a-zA-Z0-9]+)")
+					reGlblName := regexp.MustCompile("(^|[\\s])var\\s([_a-zA-Z][_a-zA-Z0-9]*)")
 
 					if match := reGlblName.FindStringSubmatch(string(line)); match != nil {
 						if _, err := prePkg.GetGlobal(match[len(match) - 1]); err != nil {

--- a/tests/main.cx
+++ b/tests/main.cx
@@ -345,7 +345,7 @@ func main ()() {
 	runTest("issue-52.cx", cx.COMPILATION_ERROR, "Invalid implicit cast.")
 	runTest("issue-53.cx", cx.COMPILATION_ERROR, "Panic when using +* in an expression")
 	runTestEx("issue-54.cx", cx.COMPILATION_ERROR, "No compilation error when defining a struct with duplicate fields.", TEST_ISSUE, 0)
-	runTestEx("issue-55.cx", cx.SUCCESS, "Can't define struct with a single character identifier.", TEST_ISSUE, 0)
+	runTest("issue-55.cx", cx.SUCCESS, "Can't define struct with a single character identifier.")
 	runTest("issue-56.cx", cx.SUCCESS, "Panic when variable used in if statement without parenthesis.")
 	runTest("issue-57.cx", cx.SUCCESS, "Struct field stomped")
 	runTestEx("issue-58.cx", cx.COMPILATION_ERROR, "No compilation error when indexing an array with a non integral var.", TEST_ISSUE, 0)

--- a/tests/testdata/tokens/main.cx.txt
+++ b/tests/testdata/tokens/main.cx.txt
@@ -2488,7 +2488,7 @@ STRLIT No compilation error when defining a struct with duplicate fields.
 INTLIT 0
 RPAREN 
 SCOLON 
- IDENT runTestEx
+ IDENT runTest
 LPAREN 
 STRLIT issue-55.cx
  COMMA 
@@ -2497,10 +2497,6 @@ PERIOD
  IDENT SUCCESS
  COMMA 
 STRLIT Can't define struct with a single character identifier.
- COMMA 
- IDENT TEST_ISSUE
- COMMA 
-INTLIT 0
 RPAREN 
 SCOLON 
  IDENT runTest

